--- a/vendor/elao/enum/src/Bridge/Symfony/Translation/TranslatableEnumTrait.php	2025-01-20 13:56:25.000000000 +0100
+++ b/vendor/elao/enum/src/Bridge/Symfony/Translation/TranslatableEnumTrait.php	2025-07-04 11:23:17.016889735 +0200
@@ -19,7 +19,7 @@
 {
     use ReadableEnumTrait;

-    public function trans(TranslatorInterface $translator, string $locale = null): string
+    public function trans(TranslatorInterface $translator, ?string $locale = null): string
     {
         return $translator->trans($this->getReadable(), [], $locale);
     }
